crates/edr_rpc_server/src/node/node_data.rs

```diff
@@ -0,0 +1,424 @@
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use indexmap::IndexMap;
+
+use edr_eth::remote::RpcClient;
+use edr_eth::signature::public_key_to_address;
+use edr_eth::{
+    remote::{BlockSpec, BlockTag, Eip1898BlockSpec},
+    Address, SpecId, U256,
+};
+use edr_evm::blockchain::{Blockchain, ForkedBlockchain, LocalBlockchain};
+use edr_evm::state::{AccountTrie, TrieState};
+use edr_evm::{
+    blockchain::{BlockchainError, SyncBlockchain},
+    mine_block,
+    state::{IrregularState, StateError, SyncState},
+    AccountInfo, Block, CfgEnv, HashMap, HashSet, MemPool, MineBlockResult, MineOrdering,
+    RandomHashGenerator, SyncBlock, KECCAK_EMPTY,
+};
```

Comment:
Merge imports

Reply:
Done in [Merge imports](https://github.com/NomicFoundation/hardhat/pull/4539/commits/0e0a2f349ef63b628abbab1547476d3a621afd6d)